apps/api/public/dashboard.html

```
@@ -770,6 +770,18 @@ <h2 class="intg-title">OAuth Applications</h2>
                                                         </div>
                                                     </div>
                                                 </label>
+                                                <label class="scope-toggle">
+                                                    <input type="checkbox" id="create-oauth-scope-pr">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)